packages/flutter_webrtc/tizen/inc/flutter_webrtc_base.h

```diff
@@ -0,0 +1,204 @@
+#ifndef FLUTTER_WEBRTC_BASE_HXX
```

Comment:
This header integrated with [flutter_common.h](https://github.com/flutter-webrtc/flutter-webrtc/blob/8f183ed17a3f0d75eb05c397bb88cc60312d7127/common/cpp/include/flutter_common.h).
Wouldn't it be better to separate the same as flutter-webrtc package for management?

Reply:
https://github.com/flutter-tizen/plugins/pull/489#discussion_r1039169899